odoo_module_migrate/migration_scripts/migrate_130_140.py

```diff
@@ -1,19 +1,153 @@
 # Copyright (C) 2020 - Today: Iván Todorovich
+# Copyright (C) 2020 - Today: Simone Rubino
 # @author: Iván Todorovich (https://twitter.com/ivantodorovich)
+# @author: Simone Rubino (https://github.com/Daemo00)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
+import re
+from pathlib import Path
+import lxml.etree as et
 from odoo_module_migrate.base_migration_script import BaseMigrationScript
 
-_TEXT_ERRORS = {
-    ".xml": {
-        "<act_window(\n|.|\t)*":
-        "act_window tag has been deprecated. Use <record> instead",
-        "<report(\n|.|\t)*":
-        "report tag has been deprecated. Use <record> instead",
+
+def src_model_new_value(field_elem, model_dot_name):
+    """
+    Particular behavior for new binding_model_id.
+
+    Apply some heuristic to change
+    old attribute `src_model=account.move`
+    to `<field name="binding_model_id" ref="account.model_account_move"/>`.
```

Comment:
:+1:

Reply:
yes I think there is no way to know what the exact `ref` will be.. This should be close enough in most cases